fdbserver/Status.actor.cpp

```diff
@@ -3478,8 +3554,8 @@ ACTOR Future<StatusReply> clusterGetStatus(
 		statusObj["clients"] = clientStatusFetcher(clientStatus);
 
 		if (configuration.present() && configuration.get().blobGranulesEnabled) {
-			JsonBuilderObject blobGranuelsStatus =
-			    wait(blobWorkerStatusFetcher(blobWorkers, address_workers, &status_incomplete_reasons));
+			JsonBuilderObject blobGranuelsStatus = wait(blobGranulesStatusFetcher(
```

Comment:
I think you need to add `timeoutError()` here to limit the time `blobGranulesStatusFetcher` takes. All actor functions in this file should have a time bound so that the client can get a timely response.

Reply:
added. Thank you